hathor/p2p/states/base.py

```diff
@@ -26,7 +27,7 @@
 
 class BaseState:
     protocol: 'HathorProtocol'
-    cmd_map: Dict[ProtocolMessages, Callable[[str], None]]
+    cmd_map: Dict[ProtocolMessages, Union[Callable[[str], None], Callable[[str], Deferred[None]]]]
```

Comment:
Arght! Really?

Reply:
It's ok to go like this. Maybe we should refactor so all methods return `Deferred[None]`.

Reply:
Some of the functions on the map will return a `Deferred`, so this is needed.

Reply:
I see. We can refactor it later. Thanks!